src/components/MovieRecommender.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+
+import useFormInput from "../hooks/useFormInput";
+import { Movie } from "../types";
+import { getRecommendations } from "../network/requests";
+
+const MovieRecommender = (props: {}): JSX.Element => {
+  const emailsInput = useFormInput("");
```

Comment:
Instead of breaking out the emails and handleChange separately here, you could just spread `{...emailsInput}` in the `<input />` further down.